src/types/index.d.ts

```diff
@@ -16,6 +22,12 @@ export interface Transformer {
    */
   name: string;
 
+  /**
+   * Whether the transformer requires user input to run
+   * If true, the transformer won't show a spinner to avoid cluttering the spinner with user prompts.
+   */
+  requiresUserInput?: boolean;
```

Comment:
This was necessary b/c otherwise the spinner would show while the user input prompt was showing and the process is waiting for user input. This would severely decrease UX because users might see the spinner as a sign that something is still loading or already going on, not making it clear that they actually need to provide input.